app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayFragment.kt

```diff
@@ -88,8 +93,21 @@ class TabsTrayFragment : AppCompatDialogFragment() {
         container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View {
-        val containerView = inflater.inflate(R.layout.fragment_tab_tray_dialog, container, false)
-        inflater.inflate(R.layout.component_tabstray2, containerView as ViewGroup, true)
+        _tabsTrayDialogBinding = FragmentTabTrayDialogBinding.inflate(
+            inflater,
+            container,
+            false
+        )
+        _tabsTrayBinding = ComponentTabstray2Binding.inflate(
```

Comment:
I wouldn't be opposed to removing the '2' in the xml class names if you're touching this code. 👀

Reply:
But then how will we know of our history when we reach component_tabstray3.xml ? 😆